### PR TITLE
commit

### PR DESCRIPTION
**Pull request by Maryna1996**

### CODE REVIEW

File: src/homework/hom20/Factory/FurnitureFactory.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package homework.hom20.Factory;
+public class FurnitureFactory {
```
Comment (kartius) [inline src/homework/hom20/Factory/FurnitureFactory.java]: It's a not an AbstractFactory. You need to make it as abstract class or interface and create a couple of factory imlementations

File: src/homework/hom20/Builder/CarBuilderImpl.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package homework.hom20.Builder;
+
+public class CarBuilderImpl implements CarBuilder {
```
Comment (kartius) [inline src/homework/hom20/Builder/CarBuilderImpl.java]: It's a not builder pattern. 
You need to return this in each method inside

File: src/homework/hom20/Builder/CarBuilder.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package homework.hom20.Builder;
+
+public interface CarBuilder {
```
Comment (kartius) [inline src/homework/hom20/Builder/CarBuilder.java]: You don't need an interface

File: src/homework/hom20/Factory/FurnitureFactory.java (hunk 1 above)
Comment (Maryna1996) [reply src/homework/hom20/Factory/FurnitureFactory.java]: corrected

File: src/homework/hom20/Builder/CarBuilderImpl.java (hunk 2 above)
Comment (Maryna1996) [reply src/homework/hom20/Builder/CarBuilderImpl.java]: corrected

File: src/homework/hom20/Builder/CarBuilder.java (hunk 3 above)
Comment (Maryna1996) [reply src/homework/hom20/Builder/CarBuilder.java]: corrected

**Review by kartius — APPROVED** [review]

**Pull request merged**